### DISCUSSION TITLE
cqhttp插件的dll

### DISCUSSION THREAD

**Discussion by haowen-li** [howmanybots/onebot#135] [问答]
注意到最近不少项目都允许使用酷q插件，如cqhttp插件，但是都要求提供dll。可能是我太菜了，我并没有找到cqhttp插件的dll，能否直接在这个项目里提供一下？

**Comment by stdrc** [comment]
在我的 coolq-http-api 项目的 release 里面下，那是原本的 CQHTTP 插件
